api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -5,6 +5,27 @@ const { lunieMessageTypes } = require('../../lib/message-types')
 
 const CHAIN_TO_VIEW_COMMISSION_CONVERSION_FACTOR = 1e-9
 
+const proposalTypeEnum = {
+  TEXT: 'TEXT',
+  COUNCIL: 'COUNCIL',
+  TREASURY: 'TREASURY',
+  PARAMETER_CHANGE: 'PARAMETER_CHANGE'
+}
+
+const proposalTypeEnumDictionary = (key) => {
```

Comment:
this looks weird. looks like individual parameters that get changed. is there a more general way to identify those proposals. or am I wrong?

Reply:
I know, but I am afraid there is no other way :shrug: What I can do is to search in the repo for the list of all individual parameters that are changeable

Reply:
Mmm, they seem to be dispersed in the code :shrug: these calls. Not so easy: https://crates.parity.io/pallet_staking/enum.Call.html#variant.set_validator_count

Reply:
that doesn't look like a proposal but the resolution of a proposal that actually changes the parameter. you are sure this is the proposal?

Reply:
this is the proposal method. It is a metadata of the proposal itself that tells what method the proposal image will execute (new code patch that will be added to the runtime once the proposal is approved)

Reply:
So what we could do is, to simplify, say if the proposal has a method then it is a Parameters Change proposal, but I am not 100% sure that the other ones have no method. Need to check. Maybe their method is something like "propose" or "plead" or something like that

Reply:
Maybe all Polkadot proposals are actually parameters change :thinking:

Reply:
OK, I defaulted to PARAMETER_CHANGE since all democracy (and therefore referendas as well) start off with this "image" concept. So there is always going to be some new chunk of code getting approved